package/mediacenter-osmc/patches/all-066-Allow-volume-actions-without-waking-up-screensaver.patch

```diff
@@ -0,0 +1,37 @@
+From 9095a8bc4f2c67fba1064d7c567b636d1ed3ce7a Mon Sep 17 00:00:00 2001
+From: Jeffrey Larson <jeffothy@gmail.com>
+Date: Mon, 8 Aug 2016 18:15:30 +0000
+Subject: [PATCH 1/1] Allow volume actions without waking up screensaver
+
+---
+ xbmc/input/InputManager.cpp | 7 +++++--
+ 1 file changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/xbmc/input/InputManager.cpp b/xbmc/input/InputManager.cpp
+index 1e21e7b..85a7615 100644
+--- a/xbmc/input/InputManager.cpp
++++ b/xbmc/input/InputManager.cpp
+@@ -589,7 +589,7 @@ bool CInputManager::OnKey(const CKey& key)
+   g_application.ResetScreenSaver();
+ 
+   // allow some keys to be processed while the screensaver is active
+-  if (g_application.WakeUpScreenSaverAndDPMS(processKey) && !processKey)
++  if (!processKey && g_application.WakeUpScreenSaverAndDPMS(processKey))
```

Comment:
Yes, changing the order of the conditions in necessary.  The original code would always call WakeUpScreenSaverAndDPMS(), so changing the order prevents calling it in the case where processKey evaluates as false.  This turned out to be the simplest cleanest patch that I could see.